psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -366,19 +366,22 @@ def scf_finalize_energy(self):
         # compute it if needed
         if core.get_option('SCF', 'REFERENCE') != "UHF":
             psio = core.IO.shared_object()
-            psio.open(constants.PSIF_SO_TEI, 1)  # PSIO_OPEN_OLD
-            try:
-                psio.tocscan(constants.PSIF_SO_TEI, "IWL Buffers")
-            except TypeError:
-                # "IWL Buffers" actually found but psio_tocentry can't be returned to Py
-                psio.close(constants.PSIF_SO_TEI, 1)
-            else:
-                # tocscan returned None
-                psio.close(constants.PSIF_SO_TEI, 1)
-                core.print_out("    SO Integrals not on disk, computing...")
-                mints = core.MintsHelper(self.basisset())
-                mints.integrals()
-                core.print_out("done.\n")
+            #psio.open(constants.PSIF_SO_TEI, 1)  # PSIO_OPEN_OLD
+            #try:
+            #    psio.tocscan(constants.PSIF_SO_TEI, "IWL Buffers")
+            #except TypeError:
+            #    # "IWL Buffers" actually found but psio_tocentry can't be returned to Py
+            #    psio.close(constants.PSIF_SO_TEI, 1)
+            #else:
+            #    # tocscan returned None
+            #    psio.close(constants.PSIF_SO_TEI, 1)
+
+            # logic above foiled by psio_tocentry not returning None<--nullptr in pb11 2.2.1
+            #   so forcibly recomputing for now until stability revamp
+            core.print_out("    SO Integrals not on disk, computing...")
```

Comment:
This is a good opportunity to remove one of the comma splices we have in our code methinks.

Reply:
In the world of quantum chemistry, someone's probably banking on that comma splice. :-) Will do.

Reply:
That's a really good point.  I guess we should be careful to avoid too many changes in output.  The most notable comma splice we had, which was probably used in greps, was the old "buy a developer a beer" banner.  Thankfully @lothian fixed that a long time ago, before it drove me nuts and affected users' grepping.

Reply:
I actually considered it more of a joke than a good point. My attitude has been that we shouldn't be tentative on changing text output — that we strive for consistent psivar access, not text access. I've already changed up a lot of `Bohr` --> `[a0]` in related branches on those grounds.